deploy/crds/integreatly_v1alpha1_postgres_cr.yaml

```diff
@@ -2,14 +2,14 @@ apiVersion: integreatly.org/v1alpha1
 kind: Postgres
 metadata:
   # name must be between 1-40 characters
-  name: example-postgres
+  name: example-postgres-man
```

Comment:
I think we can remove these changes to the CR

Reply:
My mistake, I did mean to revert them